packaging/rpm/crc.spec.in

```diff
@@ -24,7 +24,7 @@ Name:           %{goname}
 Release:        1%{?dist}
 Summary:        CRC's main executable
 License:        APL 2.0
-ExcludeArch:    armv7hl i686
+ExcludeArch:    armv7hl i686 aarch64 ppc64le
```

Comment:
I'd move this after the next commit..

Reply:
you mean first commit should be enable arm bits and then this one?

Reply:
Yes, it's odd for me to first remove `aarch64`, and then readd it in the commit right after

Reply:
Done.